pennylane/qchem/convert.py

```diff
@@ -801,8 +822,8 @@ def _rccsd_state(ccsd_solver, tol=1e-15):
     >>> mycc = cc.CCSD(myhf).run()
     >>> wf_ccsd = _rccsd_state(mycc, tol=1e-1)
     >>> print(wf_ccsd)
```

Comment:
```suggestion
    >>> wf_ccsd
```

Reply:
Doesn't print for me without the print statement